src/test/java/com/wikia/webdriver/testcases/mercurytests/HeaderTest.java

```diff
@@ -0,0 +1,123 @@
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class HeaderTest extends NewTestTemplate {
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithoutInfobox")
+  public void mercury_header_checkElementsVisibilityWithoutInfobox() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.NO_INFOBOX)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is not visible");
+    PageObjectLogging.log("hero image", MercuryMessages.INVISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isHeroImageVisible(), "hero image is visible");
+    PageObjectLogging.log("hero image", MercuryMessages.VISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_2)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is not visible");
+    PageObjectLogging.log("hero image", MercuryMessages.INVISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_heroImageIsProperlyStyled")
+  public void mercury_header_heroImageIsProperlyStyled() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    String property = "background-color";
+    String value = header.getHeroImageCssValue(property);
+    Assertion.assertEquals(
+        value,
+        "rgba(255, 255, 255, 1)",
+        "hero image has proper background-color attribute"
+    );
+    PageObjectLogging.log(String.format("header's %s", property), value, true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header's %s" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)